egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -259,6 +259,10 @@ def train(args, run_opts, background_process_handler):
     num_jobs = common_lib.get_number_of_jobs(args.tree_dir)
     feat_dim = common_lib.get_feat_dim(args.feat_dir)
     ivector_dim = common_lib.get_ivector_dim(args.online_ivector_dir)
+    num_cmn_offsets = train_lib.get_num_cmn_offsets(args.feat_dir)
```

Comment:
I think it would be a good idea to create a file 'num_cmn_offsets' in the iVector dir containing the number of these cmn offsets (or make sure it doesn't exist if the offsets are not being used).. then you can check that it's the same as in the feat dir.  A mismatch would cause hard-to-understand errors if not caught at this stage.